app/src/org/commcare/connect/ConnectManager.java

```diff
@@ -255,24 +274,12 @@ public static boolean shouldShowConnectButton() {
     }
 
     public static boolean isConnectTask(int code) {
-        return ConnectTask.isConnectTaskCode(code);
+        return true;
     }
 
     public static void handleFinishedActivity(int requestCode, int resultCode, Intent intent) {
-        if(ConnectIdWorkflows.handleFinishedActivity(requestCode, resultCode, intent)) {
             getInstance().connectStatus = ConnectIdStatus.Registering;
-        }
-    }
-    private static void completeSignin() {
-        ConnectManager instance = getInstance();
-        instance.connectStatus = ConnectIdStatus.LoggedIn;
-
-        scheduleHearbeat();
-        CrashUtil.registerConnectUser();
 
-        if(instance.loginListener != null) {
```

Comment:
If this is no longer necessary then CommCareSetupActivity shouldn't be passing in a callback handler